domains/ameen.json

```diff
@@ -0,0 +1,11 @@
+{
+    "description": "My personal portfolio website",
+    "repo": "https://github.com/am333n/my_portfolio",
+    "owner": {
+        "username": "am333n",
+        "email": "muhammedameen752@gmail.com"
+    },
+    "record": {
+        "CNAME": "https://am333n.github.io/my_portfolio/"
```

Comment:
CNAME should be just a hostname with no protocols or URL paths. Reopen when fixed.